packages/design-system-tokens/src/lib/exportCssVars.ts

```diff
@@ -133,15 +132,29 @@ const writeCssFile = (filename: string, importedModule: any, sep: string) => {
  * @returns An exit code based on success writing output
  */
 export const exportCssVars = (fileDescriptors: FileDescriptor[], outPath: string): number => {
-  fileDescriptors.forEach((file) => {
-    const importedModule = require(`${file.moduleImportName}`);
-    // component files do not need a separator
-    const sep = file.baseName.includes('component') ? '' : '-';
-
-    const layoutFilename = `${outPath}/${file.baseName}-layout-tokens.scss`;
-    const cssVarFilename = `${outPath}/${file.baseName}-theme.css`;
-    writeSassLayout(layoutFilename, file, importedModule, sep);
-    writeCssFile(cssVarFilename, importedModule, sep);
+  const groupByTheme = fileDescriptors.reduce((acc: Record<string, any[]>, file) => {
+    const theme = file.baseName.split('-')[0];
+    if (!acc[theme]) {
+      acc[theme] = [];
+    }
+    acc[theme].push(file);
+    return acc;
+  }, {});
+
+  Object.entries(groupByTheme).forEach(([theme, files]) => {
+    let output = '';
+
+    files.reverse().forEach((file) => {
```

Comment:
Rather than relying on knowing the order and reversing it, could we use some kind of string matching to make sure the one we want to come first comes first?

Reply:
I like the structure of this `exportCssVars` function!

Reply:
i actually don't think the `reverse()` matters since it's just assigning the tokens to the :root and we don't have tokens dependent on other tokens in this setup. so i removed it.